pkgs/tools/inputmethods/fcitx5/fcitx5-mozc.nix

```diff
@@ -0,0 +1,119 @@
+{ lib, clangStdenv, fetchFromGitHub, fetchurl, fetchpatch, fetchgit
+, python3Packages, mesa, ninja, pkg-config, protobuf, zinnia, qt5, fcitx5
+, jsoncpp, gtest, which, gtk2, unzip }:
+let
+  inherit (python3Packages) python gyp six;
+  japanese_usage_dictionary = fetchFromGitHub {
+    owner = "hiroyuki-komatsu";
+    repo = "japanese-usage-dictionary";
+    rev = "e5b3425575734c323e1d947009dd74709437b684";
+    sha256 = "0pyrpz9c8nxccwpgyr36w314mi8h132cis8ijvlqmmhqxwsi30hm";
+  };
+  # abseil-cpp in nixpkgs is too old
+  abseil-cpp = fetchFromGitHub {
+    owner = "abseil";
+    repo = "abseil-cpp";
+    rev = "20200923.2";
+    sha256 = "G+wkaC4IPtyc/xCUyVFJOcHppPFU7KkhIHjv6uhVKGU=";
+  };
+  zipcode_rel = "202011";
+  jigyosyo = fetchurl {
+    url =
+      "https://osdn.net/projects/ponsfoot-aur/storage/mozc/jigyosyo-${zipcode_rel}.zip";
+    sha256 = "j7MkNtd4+QTi91EreVig4/OV0o5y1+KIjEJBEmLK/mY=";
+  };
+  x-ken-all = fetchurl {
+    url =
+      "https://osdn.net/projects/ponsfoot-aur/storage/mozc/x-ken-all-${zipcode_rel}.zip";
+    sha256 = "ExS0Cg3rs0I9IOVbZHLt8UEfk8/LmY9oAHPVVlYuTPw=";
+  };
+  breakpad = fetchgit {
+    url = "https://chromium.googlesource.com/breakpad/breakpad";
+    rev = "216cea7bca53fa441a3ee0d0f5fd339a3a894224";
+    sha256 = "0Zokz1tszcnvFV4jYRQYvpdXs6ZHIUv5QodKSQJJVdc=";
+  };
+
+in clangStdenv.mkDerivation rec {
+  name = "fcitx5-mozc-${version}";
+  version = "2.26.4220.102";
+
+  src = fetchFromGitHub {
+    owner = "fcitx";
+    repo = "mozc";
+    rev = "1882e33b61673b66d63277f82b4c80ae4e506c10";
```

Comment:
Please fetch tags or change the version number to unstable-XXXX-XX-XX.

Reply:
mozc doesn't create releases or tags, instead, it stores version number in https://github.com/fcitx/mozc/blob/fcitx/src/data/version/mozc_version_template.bzl. You may find the same pattern in the package `fcitx-mozc`.